src/containers/Home/index.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import IndexHeader from 'components/header/IndexHeader';
+import {NavLink} from 'fluxible-router';
+import {connectToStores, provideContext} from 'fluxible-addons-react';
+import SurahsStore from 'stores/SurahsStore';
+import UserStore from 'stores/UserStore';
+import debug from 'utils/Debug';
+
+const styles = require('./style.scss');
```

Comment:
i think it's a ninja

On Thu, Mar 17, 2016 at 12:30 AM Mohamed El Mahallawy <
notifications@github.com> wrote:

> In src/containers/Home/index.js
> <https://github.com/quran/quran.com-frontend/pull/212#discussion_r56427411>
> :
>
> > @@ -0,0 +1,125 @@
> > +import React from 'react';
> > +import Helmet from 'react-helmet';
> > +import IndexHeader from 'components/header/IndexHeader';
> > +import {NavLink} from 'fluxible-router';
> > +import {connectToStores, provideContext} from 'fluxible-addons-react';
> > +import SurahsStore from 'stores/SurahsStore';
> > +import UserStore from 'stores/UserStore';
> > +import debug from 'utils/Debug';
> > +
> > +const styles = require('./style.scss');
>
> ?? shooting star?
>
> —
> You are receiving this because you are subscribed to this thread.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/quran/quran.com-frontend/pull/212/files/8040292bf75a8f2a16cf8c1a1b1f9450e717927e#r56427411>
>